ui/packages/consul-ui/app/components/topology-metrics/index.hbs

```diff
@@ -54,7 +54,7 @@
      {{#if @metricsHref}}
       <a class="metrics-link" href={{@metricsHref}} target="_blank" rel="noopener noreferrer" data-test-metrics-anchor>Open metrics Dashboard</a>
     {{else}}
-      <a class="config-link" href="{{env 'CONSUL_DOCS_URL'}}/agent/options.html#ui_config" target="_blank" rel="noopener noreferrer">Configure metrics dashboard</a>
+      <a class="config-link" href="{{env 'CONSUL_DOCS_URL'}}/connect/observability/ui-visualization#configuring-dashboard-urls" target="_blank" rel="noopener noreferrer">Configure metrics dashboard</a>
```

Comment:
Should we link to the top-level `/connect/observability/ui-visualization` without the anchor? Worried that that could change easily.

Reply:
Yes, on second thought I do think its better to link to the main page without the anchor.